test/hotspot/jtreg/serviceability/jvmti/HeapMonitor/MyPackage/HeapMonitor.java

```diff
@@ -164,7 +164,15 @@ public static void freeStorage() {
     enableSamplingEvents();
     setSamplingInterval(0);
 
-    // Loop around an allocation loop and wait until the tlabs have settled.
+    // Trigger GC then loop around an allocation loop and wait until Object Sampling
+    // is enabled for every later allocation. It takes two steps:
+    // 1. Consume current TLAB, whose size can be varies with heap/GC configuration
```

Comment:
"whose size can vary"

Reply:
sorry for so many typo and spell issues.

Reply:
Fixed